harvard-lts/jstor_aspace_transformer
# Ss 136

**Transformer component in tempate form**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/SS-136)

* Parent Jira Ticket [(link)](https://jira.huit.harvard.edu/browse/SS-133)
* Related Jira Ticket [(link)](https://jira.huit.harvard.edu/browse/SS-137)
* Related Jira Ticket [(link)](https://jira.huit.harvard.edu/browse/SS-138)

# What does this Pull Request do?
Initial checkin including these features
- celery and rabbit mq connectivity
- listener and do task model
- flask
- local docker


# How should this be tested?

Bring up this component, along with harvester and publisher components
See https://github.com/harvard-lts/jstor_aspace_oai_harvester/#readme on running test
Additionally run itest

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests? Yes, as part of suite with harvester and publisher

# Interested parties
Tag (@ mention) interested parties
